docs/concepts/pools/managed.md

```diff
@@ -71,6 +71,71 @@ uint256[] memory tokenWeights = _managedPool.getNormalizedWeights();
 ) = _managedPool.getGradualWeightUpdateParams();
 ```
 
+## Adding and Removing Tokens
+Pool `owner`s can edit the basket of assets in a Managed Pool by adding and removing tokens. 
+
+### Adding Tokens
+`addToken` adds a token to the Pool's list of tradeable assets. When adding a token to a Managed Pool, the weights of all other tokens in the pool will decrease. Once the new token is added, it will have an initial balance of 0. Because regular join operations do not work with tokens whose balances are 0, it is the `owner`s responsibility to deposit an initial amount of tokens into the pool via an `assetManager`. The `assetManager` is set by the `owner` when adding a token to the pool. Token additions are forbidden during weight changes or when a weight change is scheduled in the future.
```

Comment:
```suggestion
`addToken` adds a token to the Pool's list of swappable assets. When adding a token to a Managed Pool, the weights of other tokens in the pool will decrease proportionally unless any are already at the minimum weight. Once the new token is added, it will have an initial balance of 0. Because regular join operations do not work with tokens whose balances are 0, it is the `owner`s responsibility to deposit an initial amount of tokens into the pool via an `assetManager`. The `assetManager` for the new token is set by the `owner` when adding it to the pool. Token additions are forbidden during weight changes or when a weight change is scheduled in the future.
```